### DISCUSSION TITLE
Is there a way to select language during a gog install?

### DISCUSSION THREAD

**Discussion by anarcho-red** [tkashkin/GameHub#563] [Q&A]
I've downloaded XCOM 2 and got it to install and run, but the game language is in french. I cant seem to be able to change it in-game so i figured i could select the language during the install, but the installer seems to be automated.

**Comment by tkashkin** [comment]
* GOG usually has installers separated by language, in this case you should be able to select the language before downloading installers.
* Some GOG games for Windows include configurators to set options like language. GameHub should be able to detect them and add them to the context menu of the game if they are listed in the `goggame-$(id).info` file. Otherwise you'll need to run the configurator in the game's Wine/Proton prefix.
* Some games can have launch parameters or environment variables to set the language.

**Reply by anarcho-red** [reply]
thanks for the quick reply! ill try doing what you suggested after i finish downloading the installer again

**Reply by anarcho-red** [reply]
update: opted to install the game by running gog galaxy instead, works fine
